backend-comparison/benches/binary.rs

```diff
@@ -15,6 +15,14 @@ impl<B: Backend, const D: usize> Benchmark for BinaryBenchmark<B, D> {
         "Binary Ops".into()
     }
 
+    fn operation(&self) -> Option<String> {
+        Some("binary".to_string())
+    }
```

Comment:
I don't think we should have the name and the operation, the name is sufficient and more general than the operation.

Reply:
I was  thinking about how we will query the database, I would rather have an operator field than using string search in a "free form" name (with 5 bench we have already inconsistency).

We could let the name be the free form field for nice naming and add more precise fields.